### PR TITLE
[InlineImage][Test] Resolve resource not found error for inline image placeholder UITest 

### PR DESCRIPTION
**Pull request by palatter**
# What

As expected we receive an error when trying to fetch the image that doesn't exist. We want to swallow that error so that it doesn't show up in the test log.

image

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by jpeng-ms — APPROVED** [review]
thanks for the fix, but it looks like linter failed

File: packages/react-composites/tests/README.md [hunk 1]
```diff
@@ -26,10 +26,12 @@ Both hermetic and live tests use [playwright](https://playwright.dev/) test fram
...
-     *  CONNECTION_STRING
-     *  TEST_DIR
-     *  SNAPSHOT_DIR
-     *  PLAYWRIGHT_OUTPUT_DIR
+```bash
+CONNECTION_STRING = "<INSERT CONNECTION STRING HERE>"
+TEST_DIR = "tests"
+SNAPSHOT_DIR = "tests/app/browser/snapshots"
+PLAYWRIGHT_OUTPUT_DIR = "temp"
+```
```
Comment (Leah-Xia-Microsoft) [inline packages/react-composites/tests/README.md:34]: Thanks for adding these to the document!

**Comment by palatter** [comment]
> thanks for the fix, but it looks like linter failed

Couldn't find an issue. I just closed and reopened the PR and it passed the 2nd time.

**Review by Leah-Xia-Microsoft — APPROVED** [review]

File: packages/react-composites/tests/browser/chat/hermetic/ImageGallery.test.ts [hunk 2]
```diff
@@ -24,18 +24,22 @@ test.describe('ImageGallery Modal tests', () => {
...
+          remoteParticipants: [TEST_PARTICIPANTS[0], TEST_PARTICIPANTS[2]],
+          localParticipantPosition: 1,
+          sendRemoteInlineImageMessage: true,
+          inlineImageUrl: 'images/inlineImage-broken.png'
+        })
+      );
+    } catch (error) {
+      // This is expected to fail because the image url is a broken link,
+      // we want to swallow this error to prevent it from logging to the console
+      await page.locator(dataUiId('SomeImageId1')).click();
```
Comment (vhuseinova-msft) [inline packages/react-composites/tests/browser/chat/hermetic/ImageGallery.test.ts:40]: Should it be in finally block instead, so we won't loose a screenshot if there is no errors?

**Review by vhuseinova-msft — APPROVED** [review]
Left a question but otherwise looks good!

File: packages/react-composites/tests/browser/chat/hermetic/ImageGallery.test.ts (hunk 2 above)
Comment (palatter) [reply packages/react-composites/tests/browser/chat/hermetic/ImageGallery.test.ts:40]: Moving to the finally block does cause the same logging issue.

**Pull request merged**